### ISSUE TITLE
Post-session: index those who did not participate

### ISSUE THREAD

**Issue by jesswinssinger** [hmc-room-draw/room-draw#31]
https://docs.google.com/document/d/1OOfGVs4Fb2Yk5RXqzVkv199xiUvUxQKvgIwjh5IvsyI/edit

**Comment by jesswinssinger** [comment]
@g-rocket Why is this blocked?

**Comment by g-rocket** [comment]
@jesswinssinger As far as I'm aware, we don't have any where near the database features needed to implement this.  If that's not the case, feel free to unblock it.

**Comment by jesswinssinger** [comment]
@g-rocket What do you mean? There's a "has_participated" boolean for students

**Comment by g-rocket** [comment]
@jesswinssinger Then I guess the database features have been added; awesome!

**Comment by g-rocket** [comment]
Duplicate of #63
